definitions/private-law/json/CaseEventToFields/SubmitAndPay/CaseEventToFields.json

```diff
@@ -109,34 +87,12 @@
     "CaseTypeID": "PRLAPPS",
     "CaseEventID": "submitAndPay",
     "CaseFieldID": "submitAndPayDownloadApplication",
-    "PageFieldDisplayOrder": 10,
-    "DisplayContext": "READONLY",
-    "PageID": 2,
-    "PageDisplayOrder": 2,
-    "FieldShowCondition": "payAgreeStatement=\"agree\""
-  },
-  {
-    "LiveFrom": "02/12/2021",
-    "CaseTypeID": "PRLAPPS",
-    "CaseEventID": "submitAndPay",
-    "CaseFieldID": "submitAndPayDownloadApplicationLinkLabel",
-    "PageFieldDisplayOrder": 11,
+    "PageFieldDisplayOrder": 8,
     "DisplayContext": "READONLY",
     "PageID": 2,
     "PageDisplayOrder": 2,
     "FieldShowCondition": "payAgreeStatement=\"agree\""
   },
-  {
-    "LiveFrom": "02/12/2021",
-    "CaseTypeID": "PRLAPPS",
-    "CaseEventID": "submitAndPay",
-    "CaseFieldID": "c100submitAndPayDownloadApplicationLinkLabel",
-    "PageFieldDisplayOrder": 12,
-    "DisplayContext": "READONLY",
-    "PageID": 2,
-    "PageDisplayOrder": 2,
-    "FieldShowCondition": "payAgreeStatement=\"agree\" AND isEngDocGen=\"Yes\""
```

Comment:
why removed this?

Reply:
is it not common for solicitor payment journey.